PowerValidatedSolutions.psm1

```diff
@@ -491,6 +491,81 @@ Function Set-vCenterPasswordExpiration {
 }
 Export-ModuleMember -Function Set-vCenterPasswordExpiration
 
+Function Set-EsxiPasswordExpirationPeriod {
+	<#
+		.SYNOPSIS
+        Set ESXi password expiration period in days
+
+        .DESCRIPTION
+		The Set-EsxiPasswordExpirationPeriod cmdlet configures the password expiration period policies on ESXi. 
+		The cmdlet connects to SDDC Manager using the -server, -user, and -password values:
+        - Validates that network connectivity and authentication is possible to SDDC Manager
+        - Validates that the workload domain exists in the SDDC Manager inventory
+        - Validates that network connectivity and authentication is possible to vCenter Server
+        - Gathers the ESXi hosts for the cluster specificed
+        - Configured all ESXi hosts in he provided cluster
+
+        EXAMPLE
+        Set-EsxiPasswordExpirationPolicy -server sfo-vcf01.sfo.rainpole.io -user administrator@vsphere.local -pass VMw@re1! -domain sfo-m01 -cluster sfo-m01-cl01 -ExpirationInDays 60
+        This example configures all ESXi hosts within the cluster named sfo-m01-cl01 of the workload domain sfo-m01
```

Comment:
Change to: "for the workload domain".

Reply:
addressed in 430834ddd05e1b61bda2cc01157674496e856c62